cli/README.md

```diff
@@ -72,6 +72,21 @@ To add user :
 run `node cli.mjs --addUser <userId>`
 example: node cli.mjs --addUser "123~A#netflix"
 
+## Downloading external git repository
```

Comment:
Let's label this for its intended use: "Downloading JSON/YAML overrides from a git repository"